### PR TITLE
Row statistics option documentation

### PR DESCRIPTION
**Pull request by drahc1R**

### CODE REVIEW

File: docs/source/profiler.rst [hunk 1]
```diff
@@ -803,6 +803,8 @@ Below is an breakdown of all the options.
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities for unique row count
```
Comment (JGSweets) [inline docs/source/profiler.rst:806]: I think we need to clarify the extra options for `unique_count` like `hashing_methods` and other props for `HLL`

**Review by JGSweets — CHANGES_REQUESTED** [review]
Just need to add the extra details around `unique_count`

File: docs/source/profiler.rst (hunk 1 above)
Comment (taylorfturner) [reply docs/source/profiler.rst:806]: yeah the nesting should be added for HLL too you'll include `seed` and the other option for HLL

**Comment by drahc1R** [comment]
saw that base option was of dict type whoops

File: docs/source/profiler.rst [hunk 2]
```diff
@@ -803,6 +803,12 @@ Below is an breakdown of all the options.
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities from UniqueCountOptions
+        * hashing_method - (String) Property to specify row hashing method ("full" | "hll")
+        * hll - (Dict) Options for alternative method of gathering unique row count
```
Comment (JGSweets) [inline docs/source/profiler.rst:808]: I think `hll` is a class `(HyperLogLog) Options`

Comment (JGSweets) [reply docs/source/profiler.rst:808]: should note:
that is active when `hll` is the selected `hashing_method`

Comment (taylorfturner) [inline docs/source/profiler.rst:808]:
```suggestion
        * hll - (HyperLogLogOptions) Options for alternative method of gathering unique row count
```

**Review by JGSweets — APPROVED** [review]

File: docs/source/profiler.rst [hunk 3]
```diff
@@ -806,12 +806,18 @@ Below is an breakdown of all the options.
...
-      * max_sample_size - (Int) The max number of samples for the data 
+      * max_sample_size - (Int) The max number of samples for the data
         labeler
     * **correlation** - option set for correlation profiling
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities from UniqueCountOptions
+        * hashing_method - (String) Property to specify row hashing method ("full" | "hll")
+        * hll - (HyperLogLogOptions) Options for alternative method of gathering unique row count (activated when "hll" is the selected hashing_method)
```
Comment (ksneab7) [inline docs/source/profiler.rst:817]:
```suggestion
        * hll - (HyperLogLogOptions) Options for alternative method of gathering unique row count (activated when `hll` is the selected hashing_method)
```

File: docs/source/profiler.rst [hunk 4]
```diff
@@ -806,12 +806,18 @@ Below is an breakdown of all the options.
...
-      * max_sample_size - (Int) The max number of samples for the data 
+      * max_sample_size - (Int) The max number of samples for the data
         labeler
     * **correlation** - option set for correlation profiling
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities from UniqueCountOptions
+        * hashing_method - (String) Property to specify row hashing method ("full" | "hll")
+        * hll - (HyperLogLogOptions) Options for alternative method of gathering unique row count (activated when `hll` is the selected hashing_method)
```
Comment (micdavis) [inline docs/source/profiler.rst:817]:
```suggestion
        * hll - (HyperLogLogOptions) Options for alternative method of estimating unique row count (activated when `hll` is the selected hashing_method)
```

File: docs/source/profiler.rst [hunk 5]
```diff
@@ -806,12 +806,18 @@ Below is an breakdown of all the options.
...
+          * seed - (Int) Used to set HLL hashing function
```
Comment (micdavis) [inline docs/source/profiler.rst:818]:
```suggestion
          * seed - (Int) Used to set HLL hashing function seed
```

File: docs/source/profiler.rst [hunk 6]
```diff
@@ -806,12 +806,18 @@ Below is an breakdown of all the options.
...
         the default labeler 
-      * max_sample_size - (Int) The max number of samples for the data 
+      * max_sample_size - (Int) The max number of samples for the data
         labeler
     * **correlation** - option set for correlation profiling
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (Boolean) Option to enable/disable functionalities from UniqueCountOptions
+        * hashing_method - (String) Property to specify row hashing method ("full" | "hll")
```
Comment (taylorfturner) [inline docs/source/profiler.rst:816]:
```suggestion
        * is_enabled - (Bool) Enables or disables options for unique row count
        * hashing_method - (String) Property to specify row hashing method ("full" | "hll")
```

**Review by taylorfturner — CHANGES_REQUESTED** [review]
small changes

File: docs/source/profiler.rst [hunk 7]
```diff
@@ -806,12 +806,19 @@ Below is an breakdown of all the options.
...
       * data_labeler_object - (BaseDataLabeler) Datalabeler to replace 
         the default labeler 
-      * max_sample_size - (Int) The max number of samples for the data 
+      * max_sample_size - (Int) The max number of samples for the data
         labeler
     * **correlation** - option set for correlation profiling
       * is_enabled - (Boolean) Enables or disables performing correlation profiling
       * columns - Columns considered to calculate correlation
     * **row_statistics** - (Boolean) Option to enable/disable row statistics calculations
+      * unique_count - (UniqueCountOptions) Option to enable/disable functionalities from UniqueCountOptions
```
Comment (taylorfturner) [inline docs/source/profiler.rst:815]:
```suggestion
      * unique_count - (UniqueCountOptions) Option to enable/disable unique row count calculations
```

**Review by taylorfturner — APPROVED** [review]

**Pull request merged**